### PR TITLE
Update gradle version

### PR DESCRIPTION
**Pull request by aliveni**
**Change description**
- Upgrade Gradle
- Update dependencies

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -196,7 +196,7 @@ repositories {
   jcenter()
 }
 
-ext["rest-assured.version"] = '4.0.0'
+ext["rest-assured.version"] = '4.3.0'
```
Comment (doncem) [inline build.gradle:199]: let's see how java8 will work. when tried with java11 it wasn't compatible even with suggested fix by library people

File: build.gradle [hunk 2]
```diff
@@ -237,8 +237,8 @@ dependencies {
   compile group: 'io.springfox', name: 'springfox-swagger2', version: versions.springfoxSwagger
   compile group: 'io.springfox', name: 'springfox-swagger-ui', version: versions.springfoxSwagger
 
-  compile group: 'uk.gov.hmcts.reform', name: 'java-logging', version: '3.0.4'
-  compile group: 'uk.gov.hmcts.reform', name: 'logging-appinsights', version: '5.1.1'
+  compile group: 'uk.gov.hmcts.reform', name: 'java-logging', version: '5.0.1'
```
Comment (doncem) [inline build.gradle:240]: hold this off as tests need to be updated which rely on sync logging. after that is solved - can even move to very latest version

File: gradle/wrapper/gradle-wrapper.properties [hunk 3]
```diff
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-5.4.1-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-6.1.1-bin.zip
```
Comment (doncem) [inline gradle/wrapper/gradle-wrapper.properties:3]: current is 6.3 ;)

File: build.gradle (hunk 2 above)
Comment (aliveni) [reply build.gradle]: reverted the changes

File: gradle/wrapper/gradle-wrapper.properties (hunk 3 above)
Comment (aliveni) [reply gradle/wrapper/gradle-wrapper.properties]: upgraded to 6.3

**Comment by doncem** [comment]
Closing this one as well @aliveni - needs another PR

**Pull request closed without merging**